spin/preview/action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
   undeploy:
     required: false
     description: 'if true, remove the preview deployment from Fermyon Cloud'  
+  key_values:
```

Comment:
Is it possible to make this behave like a map? Thinking that it might be nice if they yaml looked like:

```
key_values:
 - abc: xyz
 - foo: bar
```

Reply:
I think Github action do not support this map format. and the workaround is to pass as follows:

```
          key_values: |-
            abc=xyz
            foo=bar
```